aws/src/database/multi_table/amend_tenure.py

```diff
@@ -43,26 +44,24 @@
 TENURE_TABLE_NAME = "TenureInformation"
 PERSONS_TABLE_NAME = "Persons"
 
-with open("input/elasticsearch_query_tenure.py", "r") as outfile:
-    ES_QUERY_CMD = outfile.read()
-
 
 def main():
-    print("== Update ==")
+    print("\n== Update ==")
     print(f"Tenure ID: {TENURE_ID}")
     print(f"Param key: {PARAM_KEY_ES}")
     print(f"New value: {NEW_PARAM_VALUE}")
-    tenure = get_tenure_dynamodb()
-    # update_elasticsearch()
+    _confirm("Correct?")
+    tenure = get_tenure_dynamodb(TENURE_ID)
+    connect_to_jumpbox_for_es(instance_id=INSTANCE_ID, stage=STAGE.value)
+    # update_tenure_elasticsearch(tenure_pk=TENURE_ID)
+    update_property_elasticsearch(property_pk=tenure["tenuredAsset"]["id"])
     # update_tenure_dynamodb(tenure)
-    update_property_dynamodb(tenure)
-    # update_person_dynamodb(tenure)
+    # update_property_dynamodb(tenure)
+    # update_persons_dynamodb(tenure)
```

Comment:
When some value needs to update/removed within a record this has to be done across multiple storages. So from my understanding that's what all these comments are for.

Any chance they could get grouped by HALO ticket type? So, for example:
``` py
def remove_tenure_end_date(tenure):
    print(f"Removing end date from tenure (id): {tenure.id}")
    print("updating tenure table")
    update_tenure_dynamodb(tenure)
    print("updating property table")
    update_property_dynamodb(tenure)
    print("updating person table")
    update_persons_dynamodb(tenure)
    print("updating elastic search...")
    etc(...)
```

Reply:
The script isn't as flexible as I'd like but it's difficult to make it flexible and keep it user-friendly and error-avoidant.

Reply:
> I'm not sure what's easiest for us to work with. I think in most cases we'd be running through every step but personally I get paranoid and like to run it bit by bit so I comment the other steps out.

Could have a "press enter to continue" or "continue? y/n" prompt before every step I suppose